lib/message_bus/client.rb

```diff
@@ -176,44 +176,50 @@ def backlog
     new_message_ids = nil
 
     @subscriptions.each do |k, v|
-      id = v.to_i
+      @bus.trace("messagebus/client/backlog/subscription") do
```

Comment:
I'm curious, why is this inside, and not outside, the `.each`? Will NewRelic provide specific insight for each run?

Reply:
Yes, it'll tell us how long they took on average and how many iterations.

Reply:
Interesting. Does it also provide max and min?